erpnext/assets/doctype/location/location.js

```diff
@@ -11,7 +11,14 @@ frappe.ui.form.on('Location', {
 			};
 		});
 	},
-
+	refresh: function (frm) {
```

Comment:
Why is this needed?
All Locations is just a label node displayed on top and each group node under All Locations would not have parent node and will be regarded as root_node which is kinda wrong. Moreover, `parent_location` isn't a mandatory field either.

Reply:
I added it because I understood that each of the nodes groups of all locations are a representation of what they contain and there should be no need to edit the node group and thus not modify the total area per group. Do you suggest to remove it?

Reply:
I remain pending the suggestion to push the changes with the solution.